src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div class="mapboxgl-map-container">
+    <div class="date is-hidden-touch">
+      <span>{{ store.maxDate }}</span>
+    </div>
+    <div class="date-mobile is-hidden-desktop">
+      <span>{{ store.maxDate }}</span>
+    </div>
+    <div ref="el" class="mapbox-container" />
+  </div>
+
+  <!-- reference image for mapbox to pull from -->
+  <img
+    ref="imageEl"
+    :src="BuoyMarker"
+    style="
+      visibility: hidden;
+      height: 15px;
+      width: 15px;
+      position: absolute;
+      top: 0;
+    "
+  />
```

Comment:
Is there a different SVG for the water markers than the buoys?

Reply:
I think the water markers are drawn with `createElement` on line 99